vendor/teleopti/calendar-data-updater.directive.js

```diff
@@ -0,0 +1,159 @@
+(function() {
```

Comment:
I understand but duplicated code is often difficult to maintain. I think we can export this as a npm module which would be easier to download from a local cache. What do you think @robink-teleopti ?